tests/contrib/django/app/views.py

```diff
@@ -23,10 +26,34 @@ def get(self, request, *args, **kwargs):
         return HttpResponse(status=403)
 
 
+def function_view(request):
+    return HttpResponse(status=200)
+
+
+class FeedView(Feed):
```

Comment:
Yes! removing the fix, will reproduce the error we had before. I could use a plain CBV with ``__call__``, but that one is a real use case